visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts

```diff
@@ -308,6 +308,25 @@ describe("codeMapMouseEventService", () => {
 		})
 	})
 
+	describe("changeCursorIndicator", ()=>{
+		it("should set the mouseIcon to grabbing, pointer, moving and default", () =>{
```

Comment:
test description is wrong.

```suggestion
it("should set the mouseIcon to grabbing", () =>{
```